src/OpenTelemetry/Resources/ResourceBuilder.cs

```diff
@@ -31,13 +31,23 @@ private ResourceBuilder()
         }
 
         /// <summary>
-        /// Creates a <see cref="ResourceBuilder"/> instance with SDK defaults
-        /// added. See <a
-        /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/">resource
+        /// Creates a <see cref="ResourceBuilder"/> instance with Default
+        /// service.name added. See <a
+        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service/">resource
         /// semantic conventions</a> for details.
         /// </summary>
         /// <returns>Created <see cref="ResourceBuilder"/>.</returns>
         public static ResourceBuilder CreateDefault()
+            => new ResourceBuilder().AddDefault();
+
+        /// <summary>
+        /// Creates a <see cref="ResourceBuilder"/> instance with Telemetry
+        /// SDK added. See <a
+        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#telemetry-sdk/">resource
+        /// semantic conventions</a> for details.
+        /// </summary>
+        /// <returns>Created <see cref="ResourceBuilder"/>.</returns>
+        public static ResourceBuilder CreateSDK()
```

Comment:
I think we should stick with the casing "Sdk" since it is used elsewhere (eg `AddTelemetrySdk`).

Reply:
Also, do we need a `CreateSdk`? How will user know which one to call (`CreateDefault()` or `CreatedSdk()`)? Probably better for us to give them just one path (pit of success).

What if we combined both into one thing?

```csharp
public static ResourceBuilder CreateDefault(bool addTelemetrySdk = true)
```

Reply:
Will fix the Sdk casing immediately. I like the idea of the optional parameter on default. This way we'll always have service.name and then optionally include the Telemetry SDK resource.

Reply:
`AddTelemetrySdk` already exists on ResourceBuilder extensions, so why need it here again?

Reply:
Is this not where we would provide access to the Telemetry Resource? We previously used CreateDefault to just link directly to AddTelemetrySDK

Reply:
Users just need an API to add TelemetrySDK, which we already provide. There is no need of an additional method.
A user can do:

ResourceBuilder.CreateEmpty().AddTelemetrySdk() - Resource will only have telemetrysdk attributes.
or
ResourceBuilder.CreateDefault().AddTelemetrySdk() - Resource will have telemetrysdk attributes + the default. (default is service.name)
or
ResourceBuilder.CreateDefault().AddTelemetrySdk().AddServiceName("CijoService") - Resource will have telemetrysdk attributes + service.name ("CijoService"). Due the merge priority, "CijoService" overrides the default one.

Reply:
This thinking makes a lot of sense to me. I'll remove the various extra things we introduced in this PR, and I like @CodeBlanch 's approach too but I think this is more elegant. Will not be implementing the optional parameter.